pkgs/applications/kde/kmail.nix

```
@@ -19,13 +58,53 @@ mkDerivation {
   };
   nativeBuildInputs = [ extra-cmake-modules kdoctools ];
   buildInputs = [
-    akonadi-search kbookmarks kcalutils kcmutils kcompletion kconfig
-    kconfigwidgets kcoreaddons kguiaddons ki18n
-    kiconthemes kinit kio kldap kmail-account-wizard kmailtransport libkdepim
-    knotifications knotifyconfig kontactinterface kparts kpty kservice
-    ktextwidgets ktnef kwidgetsaddons kwindowsystem kxmlgui libgravatar
-    libksieve mailcommon messagelib pim-sieve-editor qtscript qtwebengine
-    kdepim-addons qtkeychain libsecret
+    akonadi-search
+    kbookmarks
+    kcalutils
+    kcmutils
+    kcompletion
+    kconfig
+    kconfigwidgets
+    kcoreaddons
+    kdepim-addons
+    kguiaddons
+    ki18n
+    kiconthemes
+    kinit
+    kio
+    kldap
+    kmail-account-wizard
+    kmailtransport
+    knotifications
+    knotifyconfig
+    kontactinterface
+    kparts
+    kpty
+    kservice
+    ktextwidgets
+    ktnef
+    kwidgetsaddons
+    kwindowsystem
+    kxmlgui
+    libgravatar
+    libkdepim
+    libksieve
+    libsecret
+    mailcommon
+    messagelib
+    pim-sieve-editor
+    qtkeychain
+    qtscript
+    qtwebengine
+    akonadi-import-wizard
+    kaddressbook
+    kleopatra
+    pim-data-exporter
   ];
+  outputs = [ "doc" "out" ];
```

```suggestion
  outputs = [ "out" "doc" ];
```

I am pretty sure doc should be the default when the attr is used.

---

The [documentation](https://nixos.org/manual/nixpkgs/stable/#sec-multiple-outputs-) doesn't really tell much about how to use `outputs`…
If anyone has a deeper understanding of this than me, this might be worth a PR… I just tried to make sense of [`pkgs/build-support/setup-hooks/multiple-outputs.sh`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/setup-hooks/multiple-outputs.sh), but I feel like I miss quite some knowledge regarding the internals…

Updated `kmail.nix` according to the suggestion.

---

The first one listed in outputs gets used by default. Most of the time it should be "out".